meson.build

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+project(
+    'orc',
+    'cpp',
+    version: '2.2.0-SNAPSHOT',
+    license: 'Apache-2.0',
+    meson_version: '>=1.3.0',
```

Comment:
Given that this is for Apache ORC 2.2.0 (Year 2026), can we start with `1.8.0` or `1.8.2`?

Reply:
Perhaps, but it's more convenient to not artificially restrict what works.

Also, Debian Trixie will not have Meson 1.8.* because it was released after Trixie entered release freeze. So it will be quite some time before a stable version of Debian could build this, unless the user manually installs Meson as well.

Reply:
In addition to what @eli-schwartz has already said, this is the same version constraint used by Apache Arrow, Apache nanoarrow, and Apache arrow-adbc projects, so I think its great to keep consistent across the projects.